### PR TITLE
Issue #86: Status card updates after changing status 

### LINKED ISSUE #86: Force page reload when status changes
e.g. when a student "Drops out", the Student Profile should automatically reload with the new status reflected.

### PR DESCRIPTION
**Pull request by nondescryptid**
Fixes #86 by using computed properties — data is now reactive, and cards will change after status is changed (e.g. clicking Drop Out + confirm will change the status card to Dropped Out, clicking "reactivate for Screening" will change the card to Screening)

### CODE REVIEW

**Review by maryanahermawan — CHANGES_REQUESTED** [review]
Tested it's working well.
Suggest to clean up
1) at store students.js: A)updateStudentStatus function is not async, B)move dispatch('getAllStudents') to first then block (second then block is not needed as the first one does not return a promise).
image
2) In ModalDroppedOut.vue: this.updateStudentStatus is not returning a promise so there is no need for .then block.
image
3) I tried adding reason, the window is chopped off the edge:
add new reason window behaving unusually

What do you think of this? Let me know if you think otherwise.

**Comment by aelbore** [comment]
i dont see any student.js included to this PR also i dont see any updated file student.js to this PR.

I think its not mandatory to put async in the function if you dont have await inside the function. 
Maybe lets refactor the promise.then to async/wait to look cleaner and easy to understand.

**Review by therealyingtong — APPROVED** [review]

File: frontend/src/components/modals/ModalDroppedOut.vue [hunk 1]
```diff
@@ -115,3 +115,11 @@ export default {
     }
 }
 </script>
+<style>
+.modal-card-body{
+    width: 640px
+}
+.modal-card-foot{
+    width: 640px
```
Comment (therealyingtong) [inline frontend/src/components/modals/ModalDroppedOut.vue]:
```suggestion
.modal-card-body{
    width: auto
}
.modal-card-foot{
    width: auto
```

**Review by therealyingtong — CHANGES_REQUESTED** [review]

File: frontend/src/components/statusCards/StatusCardDroppedOut.vue [hunk 2]
```diff
@@ -43,7 +43,7 @@ export default {
     }
   },
   methods: {
-    ...mapActions([ 'updateStudentStatus' ]),
+    ...mapActions([ 'updateStudentStatus']),
```
Comment (therealyingtong) [inline frontend/src/components/statusCards/StatusCardDroppedOut.vue]: (just nitpicking here)

```suggestion
    ...mapActions(['updateStudentStatus']),
```

**Review by lauratsui — APPROVED** [review]
LGTM, pending the requested changes that have already been raised

**Comment by nondescryptid** [comment]
hi @maryanahermawan and @therealyingtong, I've just made the requested changes! please let me know if it works better now / if the modal still gets cut off. thank you for your comments!

**Review by dzldebby — APPROVED** [review]
Tested and modal doesn't cut off. Status card is reactive, works well!

**Review by therealyingtong — APPROVED** [review]

**Pull request merged**